generator/src/test/java/com/scottlogic/deg/generator/cucumber/AnyOf.feature

```diff
@@ -294,3 +294,25 @@ Scenario: Running an 'anyOf' request that contains an invalid nested allOf reque
   """
   Then I am presented with an error message
   And no data is created
+
```

Comment:
This is a good test, however it might be worth also running a test for multiple fields and an if condition relating to the foo field.

Reply:
Agreed, additional test scenarios need to be considered.